packages/fast-components-react-base/src/dialog/dialog.tsx

```diff
@@ -78,10 +89,11 @@ class Dialog extends Foundation<DialogHandledProps, DialogUnhandledProps, {}> {
      */
     public componentDidUpdate(prevProps: Partial<DialogHandledProps>): void {
         if (canUseDOM()) {
-            if (!prevProps.onDismiss && this.props.onDismiss) {
-                window.addEventListener("keydown", this.handleWindowKeyDown);
-            } else if (prevProps.onDismiss && !this.props.onDismiss) {
-                window.removeEventListener("keydown", this.handleWindowKeyDown);
```

Comment:
Are we pulling this logic for a reason?

Reply:
Restored similar logic to not listen when we don't need to